app/response_processor.py

```diff
@@ -102,6 +108,33 @@ def _encode(self, decrypted):
             raise QuarantinableError
         return xml
 
+    def _send_rm_receipt(self, case_id):
+        request_url = settings.RM_SDX_GATEWAY_URL
+
+        try:
+            r = self.session.post(request_url, auth=settings.BASIC_AUTH, json={'caseId': case_id})
+        except MaxRetryError:
+            logger.error("Max retries exceeded (5)",
+                         request_url=request_url,
+                         case_id=case_id)
+            raise RetryableError
+
+        if r.status_code == 200 or r.status_code == 201:
+            logger.info("RM sdx gateway receipt creation was a success",
```

Comment:
I know this is what we do in the consumer service but the SDX receipt service 
API states a only 201 is returned:

https://github.com/ONSdigital/rm-sdx-gateway/blob/master/API.md

It might be better to log anything else in the 200 range as a warning and retry

Reply:
i'll check for 201 and anything else can flow through to the else